Assets/Scripts/Player/Movement/PlayerMovement.cs

```diff
@@ -4,18 +4,32 @@ public sealed class PlayerMovement : MonoBehaviour
 {
     [SerializeField] private CharacterController characterController;
     [SerializeField] private float movementSpeed;
+    [SerializeField] private float gravity;
+    [SerializeField] private float groundDistance;
+    [SerializeField] private Transform groundCheck;
+    [SerializeField] private LayerMask groundMask;
     
     private Vector3 _moveDirection;
+    private Vector3 _velocity;
     private float _horizontal;
     private float _vertical;
+    private bool _isGrounded;
 
     private void Update()
     {
+        _isGrounded = Physics.CheckSphere(groundCheck.position, groundDistance, groundMask);
```

Comment:
```suggestion
        _isGrounded = Physics.CheckSphere(transform.position, groundDistance, groundMask);
```
Je kan het vanaf de pivot point van de speler van maken, hoeft niet perse. Aan jou de keus. Dan kan je de Transform varible ook weg halen.